icons/water-polo.svg

```diff
@@ -0,0 +1,15 @@
+<svg
+  xmlns="http://www.w3.org/2000/svg"
+  width="24"
+  height="24"
+  viewBox="0 0 24 24"
+  fill="none"
+  stroke="currentColor"
+  stroke-width="2"
+  stroke-linecap="round"
+  stroke-linejoin="round"
+>
+  <path d="M16.259 3.74c-.96-.96-3.6.18-5.94 2.52s-3.48 4.98-2.52 5.94 3.6-.18 5.94-2.52c2.28-2.34 3.42-4.98 2.52-5.94" />
+  <path d="M2 19c.6.5 1.2 1 2.5 1 2.5 0 2.5-2 5-2 2.6 0 2.4 2 5 2 2.5 0 2.5-2 5-2 1.3 0 1.9.5 2.5 1" />
+  <circle cx="12" cy="8" r="6" />
```

Comment:
The ball has gap violations, see https://lucide.dev/guide/design/icon-design-guide#_8-distinct-elements-must-have-2-pixels-of-spacing-between-each-other